pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -70,9 +70,41 @@ func SetDefaults_ControllerManagerConfiguration(obj *ControllerManagerConfigurat
 
 	if obj.Controllers.CloudProfile == nil {
 		obj.Controllers.CloudProfile = &CloudProfileControllerConfiguration{
-			ConcurrentSyncs: 5,
+			ConcurrentSyncs:               5,
+			KubernetesVersionManagement:   &KubernetesVersionManagement{Enabled: false},
```

Comment:
Shouldn't there be a default value set for `SupportedKubernetesVersions` and the other fields just like in the `else` case?

Reply:
The other fields are only used during the VersionManagement - so if it is not enabled, these values are not being read.